deployment/ansible/env-example/vars.yml

```diff
@@ -8,6 +8,8 @@ app_footer: © Ikatan Alumni TOKI
 nginx_domain_judgels_client: judgels.com
 nginx_domain_judgels_server_api: api.judgels.com
 nginx_domain_judgels_server_admin: admin.judgels.com
+nginx_domain_judgels_server_admin_auth_basic_user: user # <-- CHANGE THIS !!!
```

Comment:
Sori2, terakhir, boleh dibikin gini gak:

1. Ini dicomment (kayak `java_opts_judgels_server` di bawah)
2. terus seluruh yang `CHANGE THIS` diratain ke kanan, jadi yang dibawah2 dikekananin :')

Reply:
Oh dibikin beneran optional ya, okay

Reply:
Resolved in d147b40d110ed37ace11bf8076ef310cdcc61ed9